notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb

```diff
@@ -0,0 +1,354 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Marking All Pixels Used In The Photometric Aperture\n",
+    "\n",
+    "Let's display the pixels in the aperture FITS extension again, but this time, let's only mark those that are used in the optimal photometric aperture (bit mask value of 2 set).  While it is possible to do this in fewer lines, we'll show more of the steps involved for clarity."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an array that will keep track of which pixels have a bitmask\n",
+    "# value of 2 set. To start, everything is set to 0 = not set.  We make it\n",
+    "# the same dimension as the aperture pixel table, for clarity.\n",
+    "bitmask2_set = np.zeros(aperture.shape)\n",
+    "# Identify which pixels have the bit mask value of 2 set.  We'll first\n",
+    "# loop over each row (i) in the 11x11 table, and then each column (j).\n",
+    "for i,row in enumerate(aperture):\n",
+    "    for j, pix in enumerate(row):\n",
+    "        # Get the bitmask as a string of zeros and ones.\n",
+    "        this_bitmask = np.binary_repr(pix)\n",
+    "        # Is the bitmask value of 2 set?  If so, the 2nd integer from the\n",
+    "        # right should be set to 1.  We'll use Python negative indexes to\n",
+    "        # access the second integer from the right.\n",
+    "        if this_bitmask[-2] == '1':\n",
+    "            # Then record this pixel as having a bitmask value of 2 set.\n",
+    "            bitmask2_set[i,j] = 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can display the aperture, but this time we will only display the bitmask2_set table we've created, where a zero means the bitmask value of 2 is NOT set (thus these pixels aren't used in the optimal photometric aperture) and a one means the bitmask value of 2 is set (and thus these pixels were used in the optimal photometric aperture)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "# Display, as an image, the 11x11 table that records the bitmask value of\n",
+    "# 2 being set.\n",
+    "cax = ax.imshow(bitmask2_set, cmap=plt.cm.YlGnBu_r, origin=\"lower\")\n",
+    "# Add a color bar.\n",
+    "cbar = fig.colorbar(cax)\n",
+    "# Add a title to the plot.\n",
+    "fig.suptitle(\"WASP-126 b Aperture - Sector 1 - Pixels Used In Phot. Ap.\")\n",
+    "plt.show()"
+   ]
```

Comment:
Suggest adding newlines before each comment line to improve readability of the cell.